src/services/expenses.ts

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+let expenses: any[] = [];
+
+const getAllExpenses = () => {
+  return expenses;
+};
+
+const getExpenseById = (expenseId) => {
```

Comment:
this file has TS extension, array of data has type, but function arguments are not typed (as well as function result)
Add types here, or change extension and write on JS, use one option 🙂